test/e2e/dynamic_provisioning_test.go

```diff
@@ -634,6 +634,39 @@ var _ = ginkgo.Describe("Dynamic Provisioning", func() {
 		}
 		test.Run(cs, ns)
 	})
+
+	ginkgo.It("should create a NFS volume on demand with mount options [kubernetes.io/azure-file] [file.csi.azure.com] [nfs]", func() {
+		skipIfTestingInWindowsCluster()
+		skipIfUsingInTreeVolumePlugin()
+
+		pods := []testsuites.PodDetails{
+			{
+				Cmd: convertToPowershellCommandIfNecessary("echo 'hello world' > /mnt/test-1/data && grep 'hello world' /mnt/test-1/data"),
+				Volumes: []testsuites.VolumeDetails{
+					{
+						ClaimSize: "100Gi",
+						MountOptions: []string{
+							"remount",
```

Comment:
remove this `remount` option, e2e test should work

Reply:
Also removed in my machine, it still didn't work.

Reply:
just remove this `remount` in this PR, not sure why there is `remount` in your env too.